Cargo.toml

```diff
@@ -9,7 +9,7 @@ borsh = "1.5.1"
 borsh-derive = "1.0.0"
 env_logger = "0.11.5"
 chrono = "0.4.38"
-libp2p = { version = "0.51.3", features = [
+libp2p = { version = "0.51.4", features = [
```

Comment:
Maybe it would make sense to use the libp2p-identity crate directly (https://docs.rs/libp2p-identity/0.2.10/libp2p_identity/) instead, since the `identity` module is deprecated according to the docs. :thinking: 

Also, since the Bootstrap node uses 0.51.3 without secp256k1, this could be the reason this doesn't work. You could try to update it there as well (https://github.com/BitcreditProtocol/Bootstrap-Node) and run a local relay node to see if it works.

Reply:
Added a PR in the bootstrap node which enables the libsecp256k1 feature. With that connecting works. Libp2p already uses libp2p_identity internally (`pub use libp2p_identity as identity;`) so as long as we have libp2p in the project adding the identity crate extra will not change anything.

Reply:
Nice :muscle: 

Ah, interesting - then their docs are a bit misleading there. :sweat_smile: